components/SearchBar/index.js

```diff
@@ -9,7 +9,7 @@ export default function SearchBar({ searchInput, onSearch }) {
     <SearchInput
       type="text"
       value={searchInput}
-      placeholder="Search exercises..."
+      placeholder="🔎 Search exercises... "
```

Comment:
Icons are always helpful - perhaps you could even replace "search" by your icon :
🔎 for exercises